cargo-crev/src/shared.rs

```diff
@@ -679,19 +685,26 @@ pub fn iter_rs_files_in_dir(dir: &Path) -> impl Iterator<Item = Result<PathBuf>>
 
 // Note: this function is very slow
 pub fn get_geiger_count(path: &Path) -> Result<u64> {
-    let mut count = 0;
-    for metrics in iter_rs_files_in_dir(path)
-        .flat_map_ok(|path| geiger::find::find_unsafe_in_file(&path, geiger::IncludeTests::No))
+    #[cfg(feature = "geiger")]
     {
-        let counters = metrics?.counters;
-        count += counters.functions.unsafe_
-            + counters.exprs.unsafe_
-            + counters.item_impls.unsafe_
-            + counters.item_traits.unsafe_
-            + counters.methods.unsafe_;
-    }
+        let mut count = 0;
+        for metrics in iter_rs_files_in_dir(path)
+            .flat_map_ok(|path| geiger::find::find_unsafe_in_file(&path, geiger::IncludeTests::No))
+        {
+            let counters = metrics?.counters;
+            count += counters.functions.unsafe_
+                + counters.exprs.unsafe_
+                + counters.item_impls.unsafe_
+                + counters.item_traits.unsafe_
+                + counters.methods.unsafe_;
+        }
 
-    Ok(count)
+        Ok(count)
+    }
+    #[cfg(not(feature = "geiger"))]
+    {
+        Err(anyhow::Error::msg("geiger feature is not enabled"))
+    }
 }
```

Comment:
Hard to read return expression under the `cfg`. My alternatives:
1. explicit return
2. two definitions under `cfg`
3. const under cfg and explicit match

Reply:
Thanks! I hope it's done in [da80909](https://github.com/crev-dev/cargo-crev/pull/812/commits/da809097759cc45c9150ab4440238a1ea0a4dd1a).